### PR TITLE
Move EIP-2481 to "Last Call"

### PR DESCRIPTION
**Pull request by poojaranjan**
The author is ready to move the proposal to the next status.

### CODE REVIEW

**Review by cburgdorf — APPROVED** [review]

**Review by axic — APPROVED** [review]

File: EIPS/eip-2481.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ eip: 2481
 title: "eth/66: request identifier"
 author: Christoph Burgdorf (@cburgdorf)
 discussions-to: https://github.com/ethereum/EIPs/issues/2482
-status: Review
+status: Last Call
+review-period-ends: 2021-08-11
```
Comment (axic) [inline EIPS/eip-2481.md:7]:
```suggestion
review-period-end: 2021-08-11
```

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2481 state from Review to Last Call

**Comment by MicahZoltu** [comment]
This shouldn't have been merged to Last Call.  It `requires: 2464` which is still in draft, and EIPs should not advance past their dependencies.

**Comment by axic** [comment]
> This shouldn't have been merged to Last Call. It requires: 2464 which is still in draft, and EIPs should not advance past their dependencies.

My bad. Also seems 2464 depends on 2364, which is also a draft. This will take quite a bit of time.

**Comment by MicahZoltu** [comment]
*Hopefully* we can get both of those moved to Review and then Last Call on a relatively short timeframe.  All 3 can move through the phases essentially together, they just shouldn't "pass" their dependencies.

**Pull request merged**